apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -77,261 +79,163 @@ export function InstructorQuestionSettings({
+                    type="text"
+                    class="form-control"
+                    id="title"
+                    name="title"
+                    value="${resLocals.question.title}"
+                    disabled
+                  />
+                  <small class="form-text text-muted">
+                    The title of the question (e.g., "Add two numbers").
+                  </small>
+                </div>
+                <div class="form-group">
+                  <label class="h5" for="qid">QID</label>
+                  ${resLocals.authz_data.has_course_permission_edit &&
+                  !resLocals.course.example_course
+                    ? html`
+                        <button
+                          type="button"
+                          class="btn btn-xs btn-secondary align-top ml-1"
+                          id="changeQidButton"
+                          data-toggle="popover"
+                          data-container="body"
+                          data-html="true"
+                          data-placement="auto"
+                          title="Change QID"
+                          data-content="${renderEjs(
+                            __filename,
+                            "<%= include('../partials/changeIdForm') %>",
+                            {
+                              id_label: 'QID',
+                              buttonID: 'changeQidButton',
+                              id_old: resLocals.question.qid,
+                              ids: qids,
+                              ...resLocals,
+                            },
+                          )}"
+                          data-trigger="click"
+                        >
+                          <i class="fa fa-i-cursor"></i>
+                          <span>Change QID</span>
+                        </button>
+                      `
+                    : ''}
+                  ${questionGHLink
+                    ? html`<a target="_blank" href="${questionGHLink}"> view on GitHub </a>`
+                    : ''}
+                  <input
+                    type="text"
+                    class="form-control"
+                    id="qid"
+                    name="qid"
+                    value="${resLocals.question.qid}"
+                    disabled
+                  />
+                  <small class="form-text text-muted">
+                    This is a unique identifier for the question. (e.g., "addNumbers")
+                  </small>
+                </div>
+                <div>
+                  <h2 class="h5">Topic</h2>
+                  <div class="row px-3">
+                    <div class="col-1">
+                      ${renderEjs(__filename, "<%- include('../partials/topic') %>", {
+                        topic: resLocals.topic,
+                      })}
+                    </div>
+                    <div class="col-auto">${resLocals.topic.description}</div>
+                  </div>
```

Comment:
How about we wrap this in a list group? I know there will only ever be one topic, but it feels like the badge + description want to be contained in something, IMO. We could do a similar treatment for tags if you like this?

<img width="920" alt="Screenshot 2024-04-19 at 14 09 02" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/25f8a8a8-3c6a-4c82-8222-8333501ab536">


```suggestion
                  <div class="list-group">
                    <div class="list-group-item d-flex align-items-center">
                      ${renderEjs(__filename, "<%- include('../partials/topic') %>", {
                        topic: resLocals.topic,
                      })}
                      <span class="ml-3">${resLocals.topic.description}</span>
                    </div>
                  </div>
```

Reply:
Updated to this here and for tags locally in [381d728](https://github.com/PrairieLearn/PrairieLearn/pull/9687/commits/381d7287d02a906cb1405663be90d4f84dc38348).